### PR TITLE
Fix Page Titles & Meta Tags

### PR DESCRIPTION
**Pull request by sknep**
## Changes proposed in this pull request:
- Uses Site name and domain, as well as description, to fill out meta tags 
- Works even if the site doesn't have a twitter acct

## security considerations
N/A

### CODE REVIEW

File: _data/site.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: >- # this means to ignore newlines
   Aenean et sapien a leo auctor scelerisque quis nec magna. Sed dictum ante a risus vehicula facilisis.
 
 # Twitter handle. Only the handle, not the URL.
-twitter: 18F
+# twitter: 18F
```
Comment (apburnes) [inline _data/site.yaml:8]: Would this make sense to put a "variable" there?

```suggestion
# twitter: <your-agency-handle>
```

File: _includes/meta.html [hunk 2]
```diff
@@ -16,19 +16,25 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <!-- Title and meta description
     ================================================== -->
-  <title>{{page.title}}</title>
-  <meta property="og:title" content="{{page.title}}" />
-  <meta name="description" content="{{page.description}}" />
-  <meta property="og:description" content="{{page.description}}" />
+  {% assign pageTitle = title | append: " | " | append: site.title  -%}
+  {%- assign pageDesc = page.description | default: site.description -%}
+  {%- assign pageUrl = site.url | append: page.url  -%}
```
Comment (apburnes) [inline _includes/meta.html:21]: Is `site.url` the base url and `page.url` the path?

Comment (sknep) [reply _includes/meta.html:21]: Yes, that's it.

File: _data/site.yaml (hunk 1 above)
Comment (sknep) [reply _data/site.yaml:8]: we don't use that convention elsewhere in the file. It doesn't seem likely to confuse people to me.

Comment (apburnes) [reply _data/site.yaml:8]: Fair. Should we change it to `pages` or `cloud-gov` to stay on branc?

**Review by Ephraim-G — DISMISSED** [review]

**Comment by sknep** [comment]
@Ephraim-G can you approve again? It said one of my commits wasn't signed so i rebased with a new commit.

File: _data/site.yaml (hunk 1 above)
Comment (sknep) [reply _data/site.yaml]: replaced with commented yaml
```# twitter: agency_name```

**Review by Ephraim-G — APPROVED** [review]

**Pull request merged**